.gitattributes

```diff
@@ -0,0 +1,14 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.c text
+*.h text
+
+# Declare files that will always have CRLF line endings on checkout.
+*.sln text eol=crlf
```

Comment:
Is this Windows specific?

Reply:
Not sure. I was having issues with github not understand line breaks in my files and this was a solution that seemed to work. I'm not on Windows, but Mac. Better way to accomplish this?

Reply:
What is `*.sln`?

Reply:
I think I copied/pasted a "solution" to my problem without fully understanding, and my .gitattributes files actually may not be necessary. I removed it and the files seem to not have any linebreak problems. Anymore. I think this is resolved.